values.yaml

```
@@ -2430,6 +2430,11 @@ persistence:
     ##
     size: 20Gi
 
+    # You can define additional selector for pvc
+    # selector:
+    #   matchLabels:
+    #     app-volume: "concourse"
```

```suggestion
    selector:
    #   matchLabels:
    #     app-volume: "concourse"
```

To match the rest of the `values.yaml`. We typically have the key uncommented but with a null/empty value. I tested rendering it and it won't render the `selector` field if it's empty like this.